apps/dashboard/src/interpret/App.tsx

```diff
@@ -35,7 +37,16 @@ export class App extends React.Component<IAppProps> {
 
   public render(): React.ReactNode {
     const dashboardProp: IExplanationDashboardProps = {
-      ...this.props.dataset,
+      ...this.props.data,
+      ...(this.props.dataset === undefined
+        ? {
```

Comment:
Currently, I am specifically making sure that certain fields `IDataset` are non-nullable (no `?`).

Question for Ilya/PMs: should we make every field within `IDataset` nullable? Or is this implementation okay?